app/workers/structured_data/process_data_job.rb

```diff
@@ -17,18 +16,31 @@ def perform(saved_claim_id)
         @claim = SavedClaim.find(saved_claim_id)
 
         relationship_type = @claim.parsed_form['relationship']&.fetch('type', nil)
+        claimant_name = @claim.parsed_form['claimantFullName']
+        claimant_address = @claim.parsed_form['claimantAddress']
+
         veteran = BipClaims::Service.new.lookup_veteran_from_mvi(@claim)
+
+        claimant = lookup_claimant(relationship_type, veteran, claimant_name, claimant_address) if veteran
       ensure
         @claim.process_attachments! # upload claim and attachments to Central Mail
 
         # veteran lookup for hit/miss metrics in support of Automation work
         StatsD.increment("#{stats_key}.success", tags: [
```

Comment:
Outside the scope of this PR - but are you aware of https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/concerns/monitoring.rb - it may be able to replace this ensure block.

Reply:
:heavy_plus_sign: :point_up: